test/spec/modules/appnexusAstBidAdapter_spec.js

```diff
@@ -1,194 +1,165 @@
 import { expect } from 'chai';
 import { spec } from 'modules/appnexusAstBidAdapter';
 import { newBidder } from 'src/adapters/bidderFactory';
-import bidmanager from 'src/bidmanager';
 
 const ENDPOINT = '//ib.adnxs.com/ut/v3/prebid';
 
-const REQUEST = {
-  'bidderCode': 'appnexusAst',
-  'requestId': 'd3e07445-ab06-44c8-a9dd-5ef9af06d2a6',
-  'bidderRequestId': '7101db09af0db2',
-  'bids': [
-    {
-      'bidder': 'appnexusAst',
-      'params': {
-        'placementId': '4799418',
-      },
-      'placementCode': '/19968336/header-bid-tag1',
-      'sizes': [
-        [728, 90],
-        [970, 90]
-      ],
-      'bidId': '84ab500420319d',
-      'bidderRequestId': '7101db09af0db2',
-      'requestId': 'd3e07445-ab06-44c8-a9dd-5ef9af06d2a6'
-    }
-  ],
-  'start': 1469479810130
-};
-
-const RESPONSE = {
-  'version': '0.0.1',
-  'tags': [{
-    'uuid': '84ab500420319d',
-    'tag_id': 4799418,
-    'auction_id': '2256922143947979797',
-    'no_ad_url': 'http://lax1-ib.adnxs.com/no-ad',
-    'timeout_ms': 2500,
-    'ads': [{
-      'content_source': 'rtb',
-      'ad_type': 'banner',
-      'buyer_member_id': 958,
-      'creative_id': 33989846,
-      'media_type_id': 1,
-      'media_subtype_id': 1,
-      'cpm': 0.500000,
-      'cpm_publisher_currency': 0.500000,
-      'publisher_currency_code': '$',
-      'client_initiated_ad_counting': true,
-      'rtb': {
-        'banner': {
-          'width': 728,
-          'height': 90,
-          'content': '<!-- Creative -->'
-        },
-        'trackers': [{
-          'impression_urls': ['http://lax1-ib.adnxs.com/impression']
-        }]
-      }
-    }]
-  }]
-};
-
 describe('AppNexusAdapter', () => {
   const adapter = newBidder(spec);
 
-  describe('request function', () => {
-    let xhr;
-    let requests;
+  describe('inherited functions', () => {
```

Comment:
This is a good test... but it's actually testing the `newBidder` implementation. It should move to `bidderFactory_spec` if it exists.

That said... nearly every test in `bidderFactory_spec` _calls_ `callBids`... so they'd all fail if it weren't there.

Reply:
Yes it can be added their as well. 

In bidderFactory_spec we are using sampleBidder. And here it tests the actual implementation of bidder. So this would test that implementation is correct and it has inherited needed functions